README.md

```diff
@@ -32,6 +32,8 @@ Example Sensu Go handler definition:
         "command": "sensu-pagerduty-handler",
         "env_vars": [
           "PAGERDUTY_TOKEN=SECRET",
+          "PAGERDUTY_DEDUP_KEY",
```

Comment:
Does this need to be equal to a value, similar to `PAGERDUTY_TOKEN=SECRET` above?

Reply:
Good catch, updated!